src/assets/images/group_widget.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
```

Comment:
* [ ] FYI if saving as SVG instead of Inkscape SVG there will be less metadata. Alternatively we could use SVGOMG.

Reply:
Non of the (editor-only) svg's have yet been optimized. I see this as a separate task.

Reply:
FYI I had checked the existing SVGs for `xmlns:inkscape`. There were none.